rs/ic_os/manifest_tool/BUILD.bazel

```diff
@@ -0,0 +1,35 @@
+load("@rules_rust//rust:defs.bzl", "rust_binary", "rust_test")
+
+package(default_visibility = ["//rs:ic-os-pkg"])
+
+DEPENDENCIES = [
+    # Keep sorted.
+    "@crate_index//:anyhow",
+    "@crate_index//:clap",
+    "@crate_index//:regex",
+    "@crate_index//:tempfile",
+    "@crate_index//:walkdir",
+]
+
+MACRO_DEPENDENCIES = [
```

Comment:
Are you planning to fill these currently empty lists? Or could you explain to me what they're used for?

Reply:
Here is an explanation of [macros](https://doc.rust-lang.org/reference/macros.html) [procedural macros](https://doc.rust-lang.org/reference/procedural-macros.html) in rust

I'm not planning to fill this list, but perhaps other manifest_tool commands will. I see other tools in rs/ic_os/ declare an empty list for MACRO_DEPENDENCIES ([like hostos_tool](https://github.com/dfinity/ic/blob/20e19f4918f527e636422f4f6117536eba8a7b57/rs/ic_os/hostos_tool/BUILD.bazel), so I followed.

Reply:
I see, thanks for the explanation!